retrofit/src/main/java/com/xmartlabs/base/retrofit/module/RestServiceModule.java

```diff
@@ -57,6 +57,6 @@ ServiceStringConverter provideStringConverter() {
   @Provides
   @Singleton
   protected HttpUrl provideBaseUrl(Context context) {
-    return HttpUrl.parse(context.getString(R.string.url_service));
+    return HttpUrl.parse("");
```

Comment:
what do you think of throwing an exception? I think it's better. @michaelEllisUy @bryant1410